pkg/externalresource/broker.go

```diff
@@ -119,8 +129,34 @@ func NewBroker(executorID, pathPrefix string, masterCli client.MasterClient) *Br
 	}
 }
 
-func (b *Broker) NewProxyForWorker(ctx context.Context, id string) (Proxy, error) {
-	p, err := newProxy(ctx, b.pathPrefix, id)
+func (b *Broker) NewProxyForWorker(
+	ctx context.Context,
+	resourceID model.ResourceID,
+	workerID model.WorkerID,
+	jobID model.JobID) (Proxy, error) {
+
+	resp, err := b.masterCli.CreateResource(ctx, &pb.CreateResourceRequest{
```

Comment:
what's the purpose of it? I suppose we can report the `allocated` to manager when executor tick.